datacatalog/samples/quickstart/create_fileset_entry_quickstart.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_fileset_entry_quickstart(client, project_id, entry_group_id, entry_id):
+
+    # [START datacatalog_create_fileset_quickstart_tag]
+    # Import required modules.
+    from google.cloud import datacatalog_v1beta1
+    from google.api_core import exceptions
+
+    # TODO(developer): Construct a Data Catalog client object.
+    # client = datacatalog_v1beta1.DataCatalogClient()
+
+    # TODO(developer): Set project_id to your
+    #  Google Cloud Platform project ID the entry will belong.
+    # project_id = "your-project-id"
+
+    # TODO(developer): Specify the geographic location where the
+    #  entry should reside.
+    # Currently, Data Catalog stores metadata in the us-central1 region.
+    location_id = "us-central1"
+
+    # TODO(developer): Set entry_group_id to the ID of the entry group
+    #  the entry will belong.
+    # entry_group_id = "your_entry_group_id"
+
+    # TODO(developer): Set entry_id to the ID of the entry to create.
+    # entry_id = "your_entry_id"
+
+    # 1. Environment cleanup: delete pre-existing data.
```

Comment:
Are developers expected to encounter pre-existing data? This step seems like something that'd be better suited for a test setup.

Reply:
First time developers run no, the intention is to show a reproducible scenario, every time developers run it, it will work, and it shows how to delete the entries in case they need it.
WDYT?  An option would be to split this part into a cleanup tutorial.

Reply:
A quickstart sample should introduce as few concepts as possible. I recommend splitting this out.

Reply:
sure, will send a separate PR for a clean up tutorial

Reply:
The cleanup logic has been removed from this PR.